src/main/java/com/dataloom/data/DataApi.java

```diff
@@ -42,6 +43,13 @@
             @Path( NAME_SPACE ) String entityTypeNamespace,
             @Path( TYPE_NAME ) String entityTypeName );
 
+    @PUT( CONTROLLER + "/" + ENTITY_DATA + "/" + NAME_SPACE_PATH + "/" + TYPE_NAME_PATH + "/" + NAME_PATH + "/" + SELECTED )
```

Comment:
I agree what you have said. I just followed line 40. I can do a refactor for all the paths if needed. I am not sure it will break the front end